src/utils/install/journal_defaults.json

```diff
@@ -275,7 +275,7 @@
             "type": "rich-text"
         },
         "value": {
-            "default": "Dear {{ article.correspondence_author.full_name }},<br/><br/>We are requesting that you make some changes to your paper. <br/>You can access your revision here: {{ do_revisions_url }} <br/><br/>Your revisions are due on {{ revision.date_due }}.<br/><br/>Regards, <br/>{{ request.user.signature|safe }}"
+            "default": "<p>Dear {{ article.correspondence_author.full_name|se_can_see_pii:article }},</p><p>We are requesting that you make some changes to your paper.</p><p>You can access your revision here: {{ do_revisions_url }}</p><p>Your revisions are due on {{ revision.date_due }}.</p><p>Regards,</p>{{ request.user.signature|safe }}"
```

Comment:
With this in the template, I am getting an invalid filter error when testing it manually, because this tag is not loaded where the email is rendered.

Reply:
> With this in the template, I am getting an invalid filter error when testing it manually, because this tag is not loaded where the email is rendered.

Are you overwriting the built in template tags in your dev settings file?

Reply:
Ah, good catch, yeah that's it.